include/deal.II/lac/vector_operations_internal.h

```diff
@@ -2112,8 +2116,8 @@ namespace internal
       {
         if (operation == VectorOperation::insert)
           {
-            cudaError_t cuda_error_code = cudaMemcpy(data.values.get(),
-                                                     v_data.values_dev.get(),
+            cudaError_t cuda_error_code = cudaMemcpy(data.values.data(),
+                                                     v_data.values.data(),
```

Comment:
It's going to be nice to clean up all of that

Reply:
Yes, see https://github.com/dealii/dealii/pull/14537/files#diff-acb1ab2a77fc1812186da8b93c1e717f14af0e93dee3b2535efe906aa67c7c95.